### PR TITLE
feat: enable compression

### PR DESCRIPTION
**Pull request by sarat1669**
Text compression is not enabled in hono.
When we run a light house score on the sample app generated, we get 95 score.
Screenshot 2024-09-26 at 16 09 15

Enabling compression along with the meta description fix in #904 gives 100/100.

### CODE REVIEW

**Comment by dai-shi** [comment]
Hi, can you add the PR description with some details?

**Comment by dai-shi** [comment]
Thanks for the suggestion!

File: packages/waku/src/cli.ts [hunk 1]
```diff
@@ -144,6 +145,7 @@ async function runStart() {
     import(pathToFileURL(path.resolve(distDir, DIST_ENTRIES_JS)).toString());
   const app = new Hono();
   app.use(contextStorage());
+  app.use(compress())
```
Comment (dai-shi) [inline packages/waku/src/cli.ts:148]: It looks like a good addition, it's not 100% certain if this won't cause any issues for all users.
So, my suggestion is to introduce a new option `--experimental-compress` and use it explicitly.
We can also enable it for the dev server too.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Can you make a change?

File: packages/waku/src/cli.ts (hunk 1 above)
Comment (sarat1669) [reply packages/waku/src/cli.ts]: Done

**Pull request merged**